.jenkins/Jenkinsfile

```diff
@@ -0,0 +1,26 @@
+docker.image('debian').withRun('-v /var/run/docker.sock:/var/run/docker.sock'){
+
+    checkout scm
+    sh 'ls'
+    sh "sudo apt-get update && sudo apt-get install -y docker-ce"
+    sh "rm go1* && wget https://storage.googleapis.com/golang/go1.15.6.linux-amd64.tar.gz"
+    sh "sudo rm -rf /usr/local/go/"
+    sh "sudo tar -C /usr/local -xzf go1.15.6.linux-amd64.tar.gz"
+
+    HOME = '/home/ubuntu/'
```

Comment:
This set of variables isn't used.

Reply:
Thanks. I have deleted some un-used variables and put all environment variables under the **environment** block